parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs

```diff
@@ -407,17 +414,19 @@ impl pallet_bridge_grandpa::Config<BridgeGrandpaRococoInstance> for Runtime {
 	type WeightInfo = weights::pallet_bridge_grandpa_bridge_rococo_grandpa::WeightInfo<Runtime>;
 }
 
-pub const ROCOCO_BRIDGE_PARA_PALLET_NAME: &str = "Paras";
-pub const WOCOCO_BRIDGE_PARA_PALLET_NAME: &str = "Paras";
 parameter_types! {
 	pub const RelayChainHeadersToKeep: u32 = 1024;
 	pub const ParachainHeadsToKeep: u32 = 64;
 	pub const MaxRequests: u32 = 64;
 
-	pub const RococoBridgeParachainPalletName: &'static str = ROCOCO_BRIDGE_PARA_PALLET_NAME;
-	pub const WococoBridgeParachainPalletName: &'static str = WOCOCO_BRIDGE_PARA_PALLET_NAME;
+	pub const RococoBridgeParachainPalletName: &'static str = "Paras";
+	pub const WococoBridgeParachainPalletName: &'static str = "Paras";
 	pub const MaxRococoParaHeadDataSize: u32 = bp_rococo::MAX_NESTED_PARACHAIN_HEAD_DATA_SIZE;
 	pub const MaxWococoParaHeadDataSize: u32 = bp_wococo::MAX_NESTED_PARACHAIN_HEAD_DATA_SIZE;
+
+	// TODO:check-parameter - setup hard-coded rewards somehow or store them on-chain?
+	pub const DeliveryRewardInBalance: u64 = 100_000;
```

Comment:
1. The constant definitely need some update - at Rialto/Millau they're just as an example. But idk what's our plan to determine the constant :) Let's probably discuss it later
2. You probably know that, but just in case...You may just change `pub const` to `pub storage` and then the governance will be able to update the "constant" value without doing a runtime upgrade

Reply:
1. yes, I just copy those from Rialto/Millau :), I added TODO here and we will comeback later here: https://github.com/paritytech/parity-bridges-common/issues/1677
2. thank you, I didnt know that, changed. By any chance, dont you have example for that or point me?

Reply:
For example here: https://github.com/paritytech/parity-bridges-common/blob/c7c6f0ce5e88b3ba9cdcd817f4286e76221b8222/bin/millau/runtime/src/rialto_messages.rs#L44-L49 . So every "storage parameter" has its own storage key (derived from the param name) and when `get()` is called on that param, it first tries to read it from storage and if it has no value there, then the default value from rust code is used.

Reply:
yes, yes, I got this, it is changed that way,
but my bad, I meant example of call "how to change storage parameter",
I see extrinsic `system / setStorage(items)`, it is it?

Reply:
yep, that's it